torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -565,6 +634,15 @@ struct GraphFuser {
           return std::make_pair(consumer->reverseIterator(), true);
         }
         if(shouldFuse(consumer, producer)) {
+          // Insert explicit aten::expand nodes to inputs of simple mappable
+          // operators to make the inputs the same size, if necessary.
+          if (isSimpleMap(consumer) && !isFusable(consumer)) {
```

Comment:
`shouldFuse` ... `!isFusible`: this is going to be really confusing for people to read.